packages/core/src/dialects/mssql/query-generator.js

```diff
@@ -305,11 +305,10 @@ export class MsSqlQueryGenerator extends MsSqlQueryGeneratorTypeScript {
   }
 
   commentTemplate(comment, table, column) {
-    return ' EXEC sp_addextendedproperty '
-        + `@name = N'MS_Description', @value = ${this.escape(comment)}, `
-        + '@level0type = N\'Schema\', @level0name = \'dbo\', '
-        + `@level1type = N'Table', @level1name = ${this.quoteTable(table)}, `
-        + `@level2type = N'Column', @level2name = ${this.quoteIdentifier(column)};`;
+    const tableName = table.tableName || table;
+    const schemaName = table.schema || 'dbo';
```

Comment:
you can normalize the table by using `this.extractTableDetails(table)`, it will handle things like using the proper default schema

Reply:
Makes sense - i've adjusted that and tested - I'll incorporate that

Reply:
That commit didn't work - my bad as I wasn't testing correctly locally. I've fixed it up now, hopefully the pipeline checks work now.